src/scripts/compareExtensions.php

```diff
@@ -0,0 +1,196 @@
+<?php
+error_reporting(E_ALL);
+
+function cmp( $a , $b ) {
+  return strcmp( $a->name, $b->name );
+}
+
+function nameFirst( $a, $b ) {
+//  echo "comparing $a with $b ";
+  if ($a == 'name') {
+    $return = -1; // if we make it 'less' it will be first in the array
+  } else if ($b == 'name') {
+    $return = 1;
+  } else {
+    $return = 0;
+  }
+//  echo "Returning $return\n";
+  return $return;
+}
+
+$HELP = <<<HERE
+This script helps you identify different extensions used in a 3rd-party wiki compared to Meza. It does not deploy Meza. It is only an informational tool.
+
+It actually does a three-way comparison between those 3rd-party "custom" extensions to whatever you've setup in
+/opt/conf-meza/public/MezaLocalExtensions.yml as your "improved" Meza.  The intention is to ensure that at the end of the day, your MezaLocalExtensions file will accurately include any and all extensions which are found on the 3rd-party wiki, but not found in Meza core.
+
+ Note that if MezaLocalExtensions.yml is not yet setup, this script will use the example
+ provided at config/core/MezaLocalExtensions.example.yml
+ (which serves as an example for what your local Extensions specification could look like.)
+
+Supply the full URL to the public wiki API as the script argument. e.g.
+`php compareExtensions.php https://freephile.org/w/api.php`
+to get a list of extensions which would need to be added locally.
+
+Sometimes, 3rd party wikis are not publicly accessible. In that case, this script can use a .json file to represent the external wiki. If the 3rd-party wiki API is access restricted, then execute a
+'?action=query&meta=siteinfo&siprop=extensions&format=json' API query in that
+environment and save the text output to a file with a .json extension.
+
+Then supply the path to that file as the argument to the script.
+HERE;
+
+/////////////////////// Configuration ///////////////////////////////////////////////////////////////////
+
+$mezaCoreExtensionsFile = '/opt/meza/config/core/MezaCoreExtensions.yml';
+$mezaCoreExtensionsURL = 'https://raw.githubusercontent.com/freephile/meza/master/config/core/MezaCoreExtensions.yml';
```

Comment:
Please update this to pull from enterprisemediawiki/meza

Reply:
Also...this only really works if you happen to be running the `master` version of Meza. I hope to keep `master` on the latest version of MediaWiki at all times, so there's no reason someone shouldn't be running Meza `master`...but it's something to think about, especially if someone is trying to transition an installation from an old server with old versions of extensions...they may be forced to run the `27.x` version of Meza.